eng/versioning/version_client.txt

```diff
@@ -93,6 +93,7 @@ com.azure:azure-cosmos-dotnet-benchmark;4.0.1-beta.1;4.0.1-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3_2-12;1.0.0-beta.1;1.0.0-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3-1_2-12;4.14.1;4.15.0-beta.1
 com.azure.cosmos.spark:azure-cosmos-spark_3-2_2-12;4.14.1;4.15.0-beta.1
+com.azure.cosmos.spark:azure-cosmos-spark_3-3_2-12;4.14.1;4.15.0-beta.1
```

Comment:
Should we start at 4.14.1 release? Also, there is no release for this package as 4.14.1 so I guess things might fail once we release this for the first time. 
cc @alzimmermsft  how do you propose we add this new module with new release versions?

Reply:
Good to know, I think we would better be consistent rather than starting a new version completely, I will update it.

Reply:
fixed.